TravelPro.css

```diff
@@ -0,0 +1,490 @@
+    margin-left: 150px;
+    margin-top:20px;
+    font-family: "Poppins";
+    overflow-x: hidden;
+    /*-ms-overflow-style: none;*/
+}
+.navbar,.banner{
+    display:flex;
+    justify-items: auto;
+}
+.navbar{
+    margin-bottom:10px;
+}
+#logo{
+    margin-right: 323px;
+}
+#options{
+    margin-top:45px;
+}
+a{
+    font-size: 16px;
+    margin-right:40px;
+    font-weight:normal;
+}
+#sign-up{
+    border:solid;
+    border-width: 1.5px;
+    padding: 10px 20px 10px 20px;
+    border-radius: 5px; 
+    width: 90px;
+    margin-right: 30px;
+}
+#languages{
+    /*margin-right:20px;*/
+    border:none;
+    margin-right:auto;
+}
+select{
+    margin-right:auto;
+}
+#languages>option{
+    font-size: small;
+    font-family: "Poppins";
+    font-weight: bold;
+    color:#585d7f;
+}
+#options{
+    font-size: small;
+}
+.banner{
+    background-image: url('./images/traveller2.png');
+    background-size: 670px 750px;
+    background-repeat:no-repeat;
+    background-position-x: 520px;
+    background-position-y: top;
+    height:750px;
+    margin-bottom: 130px;
+}
+#left{
+    margin-top:80px;
+    padding-right:470px;
+    margin-right:20px;
+}
+#left>p{
+    margin-top:20px;
+    margin-bottom:30px;
+}
+#sub_line{
+    color:#db6952;
+    font-weight: 500px;
+    font-size: 20px;
+    margin-bottom: 10px;
+}
+#banner_text{
+    font-size:83px;
+    padding-right:10px;
+    line-height:1.1;
+    font-weight: bold;
+    color:#181E4B;
+    letter-spacing: -3px;
+}
+#sub_text{
+    font-size:16px;
+    padding-right: 200px;
+    color: #585d7f;
+    line-height:1.6;
+}
+#left>button{
+    background-color: #F1A501;
+    color:white;
+    border:none;
+    height:60px;
+    width:170px;
+    padding: 10 10 10 10;
+    border-radius: 10px;
+    font-family: "Poppins";
+    font-size:17px;
+    margin-top:10px;
+}
+.services-heading{
+    margin-top: 0px;
+    text-align: center;
+    color:#585d7f;
+    font-weight: bold;
+    font-size:18px;
+    position:relative;
+    margin-bottom:80px;
+}
+#offer{
+    margin-top: 5px;
+    font-size:53px;
+    font-weight:bold;
+    color:#181E4B;
+}
+.services-heading>img{
+    position:absolute;
+    right:-80px;
+    top:-15px;
+}
+.services-block {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 130px;
+    margin-right: 25px;
+    margin-left:20px;
+}
+.service {
+    width: 21%; 
+    padding: 5px;
+    text-align: center;
+    height: 260px;
+}
+.service:hover{
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition:0.3s;
+    border-radius: 15px;
+}
+.service-description{
+    justify-content:baseline;
+    padding: 5px 5px 5px 5px;
+    color:#282754;
+}
+.service-description>p{
+    color:#585d7f;
+    padding: 15px 15px 15px 15px;
+    margin-top:5px;
+    margin-bottom:5px;
+}
+.service-description>h3{
+    margin-top:5px;
+    margin-bottom:5px;
+}
+.destination-heading{
+    text-align:center;
+    margin-bottom:30px;
+}
+.destination-heading>p{
```

Comment:
For font-weight, ensure the correct values are used (e.g., normal, bold, 500, etc.). The value 200px and normal bold in .destination-heading>p and #heading seem incorrect.